src/Simplex/Chat.hs

```diff
@@ -1397,7 +1445,7 @@ mkChatItem :: MsgDirectionI d => ChatDirection c d -> ChatItemId -> CIContent d
 mkChatItem cd ciId content quotedItem sharedMsgId itemTs createdAt = do
   tz <- getCurrentTimeZone
   let itemText = ciContentToText content
-      meta = mkCIMeta ciId itemText ciStatusNew sharedMsgId tz itemTs createdAt
+      meta = mkCIMeta ciId itemText ciStatusNew sharedMsgId False False tz itemTs createdAt
```

Comment:
I don't think False's need to be passed, there is no case you need it set to True

Reply:
there are store methods